internal/lm/nvml.go

```diff
@@ -101,11 +101,21 @@ func newVersionLabeler(manager resource.Manager) (Labeler, error) {
 	}
 
 	labels := Labels{
+		// Deprecated labels
 		"nvidia.com/cuda.driver.major":  driverMajor,
 		"nvidia.com/cuda.driver.minor":  driverMinor,
 		"nvidia.com/cuda.driver.rev":    driverRev,
 		"nvidia.com/cuda.runtime.major": fmt.Sprintf("%d", *cudaMajor),
 		"nvidia.com/cuda.runtime.minor": fmt.Sprintf("%d", *cudaMinor),
+
+		// New labels
+		"nvidia.com/cuda.driver-version.major":    driverMajor,
+		"nvidia.com/cuda.driver-version.minor":    driverMinor,
+		"nvidia.com/cuda.driver-version.revision": driverRev,
+		"nvidia.com/cuda.driver-version.full":     driverVersion,
+		"nvidia.com/cuda.runtime-version.major":   fmt.Sprintf("%d", *cudaMajor),
+		"nvidia.com/cuda.runtime-version.minor":   fmt.Sprintf("%d", *cudaMinor),
+		"nvidia.com/cuda.runtime-version.full":    fmt.Sprintf("%d.%d", *cudaMajor, *cudaMinor),
```

Comment:
Let's add an "always zero" `cuda.runtime-version.revision` aswell.

Reply:
Got it. So do we know for certain that cuda versions will have a `patch` number in their semver ?

Reply:
I'm happy to not included it though, if you feel that this will be confusing.

Reply:
If there is no #define declaration for the CUDA revision number, then I am not sure if there is any value adding an `always zero` label

Reply:
Yes, let's not add the `.revision` label here then.

Reply:
Got it. Can we resolve this comment?